tools/buildsteps/defaultenv

```diff
@@ -41,8 +41,8 @@ case $XBMC_PLATFORM_DIR in
     ;;
 
   android)
-    DEFAULT_NDK_VERSION="20"
-    DEFAULT_NDK_API="21"
+    DEFAULT_NDK_VERSION="20" # NDK package version (newer API can be inside)
```

Comment:
Would it make sense to increase this to 21, Android only offers the "android-ndk-r21" and and 20 no longer supports?

Do not know if the 21 would already be available on our Jenkins.

Reply:
Try it?